Sources/CardPayments/Models/Card.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 /// Represents raw credit or debit card data provided by the customer.
-public struct Card {
+public struct Card: Codable {
```

Comment:
This should only need to be `Encodable`. IMO using only `Encodable` and `Decodable` when needed helps readability.

Reply:
Addressed in 8f05559e036893aa08c529ea9d5f2534df81d234